app/code/Transoft/Blog/Controller/Adminhtml/Index/Index.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Controller\Adminhtml\Index;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\View\Result\PageFactory;
+
+/**
+ * Admin index controller
+ */
+class Index extends Action implements HttpGetActionInterface
+{
+    /**
+     * @var bool|PageFactory
+     */
+    private $resultPageFactory;
+
+    /**
+     * Constructor.
+     *
+     * @param Context $context
+     * @param PageFactory $resultPageFactory
+     */
+    public function __construct(
+        Context $context,
+        PageFactory $resultPageFactory
+    ) {
+        parent::__construct($context);
+        $this->resultPageFactory = $resultPageFactory;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function execute()
+    {
+        return $this->resultPageFactory->create();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function _isAllowed()
+    {
+        return $this->_authorization->isAllowed('Transoft_Blog::blog_manage_items');
```

Comment:
Just change \Magento\Backend\App\AbstractAction::ADMIN_RESOURCE value in your action. No need to override \Magento\Backend\App\AbstractAction::_isAllowed

Reply:
fixed